module-dev-docker-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+# Install dependencies for all local dev modules
+
+if [ -d /app/module-local-dev ]; then
+  echo "Installing dependencies for all local dev modules..."
+  for module in $(ls /app/module-local-dev/); do
+    cd /app/module-local-dev/$module
+    if [ -d node_modules ]; then
+      echo "Skipping installation of module $module dependencies, to force, remove its node_modules directory"
+    else
+      echo "Installing dependencies for module $module"
+      yarn install --prod
+    fi
+  done
+fi
+
+exec "$@"
```

Comment:
Will this be run in the correct directory?
There are some cd being done above, so the cwd will be `/app/module-local-dev/something`.
So will the `exec ./headless_ip.js` work as expected?

Reply:
Good observation. The cd works for multiple dependencies because they are absolute. The exec replaces the process including its working directory so yes the headless still works. 
I tried it on my machine with no issues and you can also double confirm.